package-name/package-name.cabal.template

```diff
@@ -13,7 +13,7 @@ extra-source-files:
     CONTRIBUTING.md
     README.md
 license-file:
-    LICENSE.txt
+    LICENSE.md
```

Comment:
I didn't see this change until after I committed d54ba48f23dc533931a243eb18dee76df9c6be4b, which does the same thing. 311da6bf6a85a859332e32d99415ccb776995919 ensures that this problem won't happen again.